src/inc/request-handler.ts

```diff
@@ -70,7 +70,11 @@ export default class RequestHandler {
       }
 
       // Refresh token before sending current request
-      await this.$auth.refreshTokens()
+      await this.$auth.refreshTokens().catch(async () => {
+        // Tokens couldn't be refreshed. Force reset.
+        await this.$auth.reset()
```

Comment:
Why reset is async? And if it makes any network calls, maybe it also can throw errors?

Reply:
Talking to @JoaoPedroAS51 seems we await everywhere but is not async. Delegating to another PR